### PR TITLE
Let GLMAlgo be less verbose

### PR DESCRIPTION
**Pull request by dbarbier**
INFO should give reasonable amount of output, user may change log level to DEBUG to have full output.

### CODE REVIEW

**Comment by regislebrun** [comment]
+1

File: lib/src/Base/Optim/NLopt.cxx [hunk 1]
```diff
@@ -348,6 +348,7 @@ void NLopt::run()
     result.store(inP, outP, absoluteError, relativeError, residualError, constraintError);
   }
 
+  result.setIterationNumber(size);
```
Comment (jschueller) [inline lib/src/Base/Optim/NLopt.cxx]: That's not a really number of iteration but the number of evaluations.

**Comment by jschueller** [comment]
These changes are fine, but maybe e4fa2b3 can be discussed a little bit ?

File: lib/src/Base/Optim/NLopt.cxx (hunk 1 above)
Comment (dbarbier) [reply lib/src/Base/Optim/NLopt.cxx]: IIRC I already requested to merge these numbers, maybe we can agree now?

See how they are used in OT, it does not make sense to define two numbers.

Comment (jschueller) [reply lib/src/Base/Optim/NLopt.cxx]: I agree one could use only one var.

But note that TNC uses getMaximumIterationNumber whereas NLopt uses getMaximumEvaluationNumber as maximum number of evaluations used by their solver.

Comment (dbarbier) [reply lib/src/Base/Optim/NLopt.cxx]: and Cobyla uses getMaximumIterationNumber() to define the maximum number of evaluations ;-)

Comment (jschueller) [reply lib/src/Base/Optim/NLopt.cxx]: So maybe NLopt could use getMaximumIterationNumber too, and we deprecated get/setMaximumIterationNumber, update the doc of getMaximumIterationNumber to mention it also applies to the number of evaluations depending on the solver, what do you think ?

Comment (jschueller) [reply lib/src/Base/Optim/NLopt.cxx]: Ok I guess it can be part of another PR

**Pull request merged**